### DISCUSSION TITLE
Unusual error

### DISCUSSION THREAD

**Discussion by merlinbest5** [biigle/core#974] [General]
One of our team members encountered an error that I haven't seen before.. The first half of the video works as normal, but after ~00:04:17 I am unable to make whole frame annotations and get the error "The frames.0 must not be greater than 258.092." The video appears to be formatted normally on my end, and the rest of the videos in the volume have been behaving normally. Any thoughts would be welcome.

![image](https://github.com/user-attachments/assets/e14fb5ba-25ff-439c-a9ca-3f31e8acebe5)

**Comment by merlinbest5** [comment]
Link here in case anyone from Biigle wants to take a look: https://biigle.de/videos/32878/annotations?x=819&y=498&r=169&t=4229065#

**Reply by mzur** [reply]
Thanks. It seems like there was some issue when the video was initially processed by BIIGLE and it thought that it was shorter than it actually is. I've fixed that manually now.
